content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
+author: bvdawson
+---
+
+*Jenkins Community Survey Results –Continuous Delivery, Pipelines and
+Star Wars*
+
+image:/images/blog/2016-survey-blog-butler.png[image,width=84,height=115]Last
+fall, at the Jenkins User Conference – US West (JUC), the community
+asked JUC attendees to take a survey. Almost 250 of you did – and thanks
+to your input, we have some interesting results about what you are doing
+with Jenkins.
+
+Back in 2012, at the time of the last community survey, 83% of you felt
+that Jenkins was mission-critical. By 2015, the percentage saying that
+Jenkins was mission-critical was 92%. Additionally, echoing the
+importance of Jenkins, 89% of respondents said their use of Jenkins had
+increased over the last year, while 11% said it had stayed the same. 0%
+said that it had decreased.
+
+The trend in the industry over the last couple of years has been to
+adopt continuous delivery (CD), thus pushing automation further down the
+pipeline – from development all the way into production. Jenkins is
+readily adaptable to this trend, since it is an orchestrator of
+automation processes, but not tied to any specific phase of software
+delivery. With its extensive plugin ecosystem, Jenkins can integrate
+with almost any technology used in any phase of software delivery.
+
+The trend to CD is clearly reflected amongst the community: 59% of
+respondents are using Jenkins for CI, but an additional 30% have
```

Comment:
Since it's not previously defined in this article, please expand the "CI" to "continuous integration (CI)", at least the first time it is referenced.